pkg/app/piped/platformprovider/ecs/client.go

```diff
@@ -417,6 +417,30 @@ func (c *client) GetListenerArns(ctx context.Context, targetGroup types.LoadBala
 	return arns, nil
 }
 
+func (c *client) GetListenerRuleArns(ctx context.Context, listenerArns []string) ([]string, error) {
+	var ruleArns []string
+
+	// 各リスナーのルールを取得
```

Comment:
Please use comment in English

Reply:
@khanhtc1202
Thank you for the feedback.
I'll make sure to use English in the comments moving forward.
[d799ab0](https://github.com/pipe-cd/pipecd/pull/4540/commits/d799ab0af2bb3920493324bcd3e6cc4c48972933)